Testing/ContinuousIntegration/AzurePipelinesMacOS.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   timeoutInMinutes: 0
   cancelTimeoutInMinutes: 300
   pool:
-    vmImage: macOS-10.15
+    vmImage: macos-11
```

Comment:
Maybe it is better to pin it to a specific sub-version, such as `macos-11.6`? We will be less at a whim of Azure Pipelines updating their versions.

Reply:
Unfortunately, I does not look like there is the possibility to specify a minor version in this case:
https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml

Reply:
Aha, I guess that Apple decided to be changing major version numbers each year now, as opposed to minor version numbers as before.